indico/core/db/sqlalchemy/protection.py

```diff
@@ -442,4 +442,6 @@ def _resolve_principal(principal):
 
 
 def render_acl(obj):
-    return jsonify_template('_access_list.html', acl=obj.protection_parent.get_access_list(skip_managers=True))
+    inherited_acl = [x for x in obj.protection_parent.get_access_list(skip_managers=True)
+                     if x not in [entry.principal for entry in obj.acl_entries]]
```

Comment:
very inefficient. you build that list for every iteration.

Do `own_entries = {entry.principal for entry in obj.acl_entries}` instead and then check against that set